views/layout.html

```diff
@@ -26,6 +26,28 @@
           <li><a href="/contact">Contact</a></li>
           <li><a href="https://github.com/FrancescoSTL/Site-Sonar"><b>Download</b></a></li>
         </ul>
+        <div class="col-sm-3 col-md-3 pull-right">
+          <form class="navbar-form" role="search" method="get" action="/search">
+            <div class="input-group">
+              <input type="text" class="form-control" placeholder="Search" name="query" id="srch-term">
+              <div class="input-group-btn">
+                <button class="btn btn-default" type="submit"><i class="fa fa-search" aria-hidden="true"></i></button>
+              </div>
+            </div>
+            <div class="input-group" id="hidden">
+              <div class="radio">
```

Comment:
well umm, this [guy](http://uxmovement.com/forms/stop-misusing-select-menus/) recommends radio buttons for selecting fewer than 5 options.
tl;dr;
<img width="692" alt="screenshot_radio" src="https://cloud.githubusercontent.com/assets/5105099/18228458/d337a772-7203-11e6-9b2a-0aba83cdfcda.png">